assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -903,10 +903,20 @@ form {
 }
 
 .button--newsletter {
-  color: #29a05f;
+  color: #ffffff;
+  background-color: #22834D;
```

Comment:
where did you get this colour from? If it is a colour from the guardian palette then we should add it to `colours.scss` and reference it

Reply:
from the link in the trello card